node/src/Pos/Block/Network/Types.hs

```diff
@@ -64,13 +64,11 @@ instance Buildable MsgGetBlocks where
 -- | 'Headers' message (see protocol specification).
 data MsgHeaders ssc
     = MsgHeaders (NewestFirst NE (BlockHeader ssc))
-    | MsgNoHeaders Text
```

Comment:
You did something wrong. This change came from `cardano-sl-1.0` but your last change removed it.